client/my-sites/people/people-list-item/index.jsx

```diff
@@ -1,36 +1,45 @@
 /**
  * External dependencies
  */
-import React from 'react';
-import PureRenderMixin from 'react-pure-render/mixin';
+import React, { PureComponent, PropTypes } from 'react';
+import { connect } from 'react-redux';
 import classNames from 'classnames';
 import omit from 'lodash/omit';
+import identity from 'lodash/identity';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import CompactCard from 'components/card/compact';
 import PeopleProfile from 'my-sites/people/people-profile';
-import analytics from 'lib/analytics';
 import config from 'config';
+import { recordGoogleEvent } from 'state/analytics/actions';
 
-export default React.createClass( {
+const browserWindow = 'undefined' === typeof window
+    ? window
+    : { scrollTo: identity };
 
-	displayName: 'PeopleListItem',
+export class PeopleListItem extends PureComponent {
+	static propTypes = {
+		translate: PropTypes.func,
+	};
 
-	mixins: [ PureRenderMixin ],
+	static defaultProps = {
+		translate: identity,
+	};
 
-	navigateToUser() {
-		window.scrollTo( 0, 0 );
-		analytics.ga.recordEvent( 'People', 'Clicked User Profile From Team List' );
-	},
+	navigateToUser = () => {
+		browserWindow.scrollTo( 0, 0 );
+		this.props.recordGoogleEvent( 'People', 'Clicked User Profile From Team List' );
```

Comment:
@markryall probably reasonable to leave this in for now, especially since this component needs other big work, but as we move towards Redux-based actions we have the opportunity to wrap UI-type actions inside of metrics tracking so that the component becomes totally unaware of the need for metrics.

Imagine, for example, that this click triggered some action, `NAVIGATE_TO_USER_PROFILE` (which in this case we don't need since all we're doing is scrolling). Instead of dispatching both the navigate action _and_ the metrics from inside the component, we can wrap `NAVIGATE_TO_USER_PROFILE` inside of `connect()` and compose them.

```js
const mapDispatchToProps = ( {
	navigateToUser: withAnalytics(
		recordGoogleEvent( 'People', 'Clicked User profile from team list' ),
		navigateToUser()
	)
} );
```

this is exciting because it keeps all of the metrics code as an incidental detail while the component only cares about doing what is intrinsically related: navigating to the users area.